pom.xml

```diff
@@ -504,7 +505,7 @@
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpcore</artifactId>
-        <version>${commons.httpclient.version}</version>
```

Comment:
He's not removing it, httpclient and httpcore used to be set at the same version number. Now they have different versions